lnwallet/channel.go

```diff
@@ -3501,7 +3501,16 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 	// as that's what they should send.
 	case msg.NextLocalCommitHeight == remoteTailHeight+1:
 		commitPoint = lc.channelState.RemoteCurrentRevocation
+
+	// Alternatively, if their height is two beyond what we know their best
+	// height to be, then they're holding onto two commitments, and the
+	// highest unrevoked point it their next revocation.
+	//
+	// TODO(roasbeef): verify this in the spec...
+	case msg.NextLocalCommitHeight == remoteTailHeight+2:
```

Comment:
this needs a unit test...

Reply:
It's really implementation specific. For example, `lnd` never holds two commitments. Even within the spec, it's unclear _which_ point to expect/use in this scenario. I'd rather add the test in a follow change (or remove it from this diff) just to keep this thing moving forwards. K